README.rst

```diff
@@ -124,12 +132,16 @@ Options
                             format
 
 
-Twine also includes a ``register`` command which is `not supported in
-Warehouse (the new PyPI software running on pypi.org)
-<https://github.com/pypa/warehouse/issues/1627>`_ and which is `no
-longer necessary if you are uploading to pypi.org
-<https://packaging.python.org/guides/migrating-to-pypi-org/#registering-package-names-metadata>`_. For
-completeness, its usage:
+Twine also includes a ``register`` command.
+
+.. WARNING:: Warning!
+   ``register`` is `not supported
+   <https://github.com/pypa/warehouse/issues/1627>`_ in `Warehouse`_ (the
+   new PyPI software running on pypi.org) and is `no longer
+   necessary if you are uploading to pypi.org
+   <https://packaging.python.org/guides/migrating-to-pypi-org/#registering-package-names-metadata>`_.
```

Comment:
This kind of makes it sound like Warehouse is at fault for not supporting this, I would tweak this a bit to say something like

> 'register' is no longer necessary when uploading to PyPI, however you may need this if you are using a some other package index

especially considering that pypi.org is the only true "PyPI" that anyone will be uploading to.

Reply:
Thanks! I am updating the wording.